requirements-dev.txt

```diff
@@ -25,3 +25,7 @@ releases==1.5.0
 
 # Release
 bumpversion==0.5.3
+
+# eth-tester
```

Comment:
do you need the commented out lines?

Reply:
hm, no, I've commented these out when testing dependencies, but since `eth-tester` updated to support most recent `py-evm` the uncommented line is correct and I'll remove the others